projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/block-editor.js

```diff
@@ -30,37 +29,77 @@ export default class BlockEditorPage extends Page {
 		return it;
 	}
 
+	//region selectors
```

Comment:
I like the overall approach. I'm kinda on the fence with naming, but I guess there is no way to make them readable AND compact